POC/GPT4V-OCR.ipynb

```diff
@@ -0,0 +1,549 @@
+{
```

Comment:
<p>I assume you are not saving the images themselves as txt files, but rather the GPT4-V OCR outputs?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BU-Spark/ml-herbarium/pull/116/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>Yes just the OCR outputs as a txt file</p>